service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2417,6 +2531,17 @@ components:
           description: The ID of the Data Repo snapshot
           type: string
 
+    UpdateGitRepoReferenceRequestBody:
+      type: object
+      properties:
+        name:
+          $ref: "#/components/schemas/Name"
+        description:
+          type: string
+        gitUrl:
```

Comment:
This name could become ambiguous if we later add support for other URLs associated with a repo, like tags, SHAs, individual files, etc.

Reply:
true. This is only for the cloning url. I updated the name